src/components/biggive-accordion/biggive-accordion.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component, Prop, Element, h } from '@stencil/core';
+import { brandColour } from '../../globals/brand-colour';
+import { spacingOption } from '../../globals/spacing-option';
+
+@Component({
+  tag: 'biggive-accordion',
+  styleUrl: 'biggive-accordion.scss',
+  shadow: true,
+})
+export class BiggiveAccordion {
+  @Element() host: HTMLBiggiveAccordionElement;
+
+  @Prop() spaceBelow: spacingOption = 0;
+
+  @Prop() textColour: brandColour = 'black';
+
+  @Prop() headingColour: brandColour = 'primary';
+
+  children: Array<HTMLBiggiveAccordionEntryElement> = [];
+
+  componentWillLoad() {
+    this.children = Array.from(this.host.children) as Array<HTMLBiggiveAccordionEntryElement>;
+  }
+
+  expandHandler(e) {
```

Comment:
This is for expanding and contracting, right? I was a bit confused reading it, would it be worth renaming to something like `expandContractHandler` maybe `toggleState`?

Also `e` is implicitly `any` here, could we make it a more specific type? I'm not sure what the type should be, maybe `MouseEvent`? But that gives the error as shown below so maybe it's wrong.

![image](https://user-images.githubusercontent.com/159481/223143345-15053d71-fc86-49d0-9d68-afcf64ca627a.png)

Reply:
Updated to use MouseEvent